cmd/root.go

```diff
@@ -34,6 +37,20 @@ var RootCmd = &ctc_lib.ContainerToolListCommand{
 the structure of a container image.
 These tests can be used to check the output of commands in an image,
 as well as verify metadata and contents of the filesystem.`,
+			PersistentPreRunE: func(cmd *cobra.Command, args []string) error {
+				if testReport != "" {
+					// Force JsonOutput
+					flags.JsonOutput = true
+					// Create TestReport File with permissions 0666 Truncate previous if Exists already.
+					TestReportFile, err := os.Create(testReport)
+					if err != nil {
+						ctc_lib.Log.Error(err)
```

Comment:
Do we need to both log and return the error? Should this just be PersistentPreRun?